services/brig/src/Brig/API/User.hs

```diff
@@ -582,8 +582,10 @@ changeSelfEmail u email allowScim = do
   changeEmail u email allowScim !>> Error.changeEmailError >>= \case
     ChangeEmailIdempotent ->
       pure ChangeEmailResponseIdempotent
-    ChangeEmailNeedsActivation (usr, adata, en) -> do
-      lift $ sendOutEmail usr adata en
+    ChangeEmailNeedsActivation (usr, adata, en) -> lift $ do
+      sendOutEmail usr adata en
+      wrapClient $ Data.updateEmailUnvalidated u email
+      Intra.onUserEvent u Nothing (UserUpdated ((emptyUserUpdatedData u) {eupEmailUnvalidated = Just email}))
```

Comment:
since we're not doing this before, I don't think we should do it now.  that's probably another reason to store this column in a separate table.  my (not very reliable) intuition also says that may be easier on cassandra resources.

Reply:
ah, I think this is also how reIndex is called?  sorry, i really should have more context.  you need to decide what you're doing, but what I think is required here is that (a) the change in unvalidated email does NOT trigger an event being sent to any other users (except, probably, team management, come to think of it); (b) re-index is called somewhere.